ui/StatusQ/src/StatusQ/Components/private/statusMessage/StatusTextMessage.qml

```diff
@@ -80,10 +80,10 @@ Item {
                                                ? 200
                                                : chatText.implicitHeight
 
-        width: parent.width
         height: effectiveHeight + d.showMoreHeight / 2
         anchors.left: parent.left
         anchors.leftMargin: d.isQuote ? 8 : 0
+        anchors.right: parent.right
```

Comment:
To be able to set right margin (for the floating Unpin button); doesn't make much sense to combine anchors+width